### PR TITLE
mark old transformation as migrated

### PR DESCRIPTION
**Pull request by ondrajodas**
Jira - https://keboola.atlassian.net/browse/COM-725

### CODE REVIEW

**Comment by ondrajodas** [comment]
@pivnicek mohl bych poprosit ještě o jedno review 🙏

File: tests/functional/migrate-simple-transformation/tearDown.php [hunk 1]
```diff
@@ -13,6 +14,12 @@
         $transformation = $manager->getTransformationV2($item);
         $result[] = $transformation['configuration'];
 
+        $oldTransformation = $manager->getTransformation($test->getTransformationBucketId());
+
+        Assert::assertArrayHasKey('configuration', $oldTransformation);
+        Assert::assertArrayHasKey('migrated', $oldTransformation['configuration']);
+        Assert::assertTrue($oldTransformation['configuration']['migrated']);
```
Comment (ondrajodas) [inline tests/functional/migrate-simple-transformation/tearDown.php:21]: kontrola zda je označená stará transformace jako zmigrvaná

**Review by pivnicek — COMMENTED** [review]
looks good.

one question, will this break the config validation of the old configuration?

**Comment by ondrajodas** [comment]
@pivnicek no, it just marks old transformation as migrated, after successfully migrate to the new transformation

**Review by pivnicek — APPROVED** [review]
ok, as long as the old configs can still be run even when they have this set to true

**Comment by ondrajodas** [comment]
@pivnicek yeah... C&P from epic:
```
* old transformation won't be killed - just marked in UI "migrated"
* old transformation can be run again even after migration
```

**Pull request merged**